tests/UI/campaigns/functional/BO/03_catalog/03_monitoring/sortAndPagination/01_emptyCategories.js

```diff
@@ -184,45 +172,65 @@ describe('Sort list of empty categories', async () => {
     });
   });
 
-  // 3 : Delete the 3 created categories
-  describe('Delete the created 3 categories from monitoring page', async () => {
-    const tests = [
-      {args: {categoryToCreate: firstCreateCategoryData}},
-      {args: {categoryToCreate: secondCreateCategoryData}},
-      {args: {categoryToCreate: thirdCreateCategoryData}},
-    ];
+  // 3 : Pagination
+  describe('Pagination next and previous', async () => {
+    it('should change the items number to 10 per page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'changeItemNumberTo10', baseContext);
+
+      const paginationNumber = await monitoringPage.selectPaginationLimit(page, tableName, '10');
+      expect(paginationNumber).to.contains(' (page 1 / 2)');
+    });
+
+    it('should click on next', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnNext', baseContext);
+
+      const paginationNumber = await monitoringPage.paginationNext(page, tableName);
+      expect(paginationNumber).to.contains(' (page 2 / 2)');
```

Comment:
Do we really care of this space?

Reply:
It's a mistake, I will fix it! thank you

Reply:
Ok np, it contains some more above ;)